csharp-basic-topics/TaskCompletedVsReturnInCSharp/TaskCompletedVsReturnInCSharp.Tests/TaskCompletedClassAndReturnClassTests.cs

```diff
@@ -0,0 +1,25 @@
+namespace TaskCompletedVsReturnInCSharpTests
+{
+    public class TaskCompletedClassAndReturnClassTests
+    {
+        [Fact]
+        public void WhenCallingUseTaskCompletedMethodAsync_ThenUseTaskCompleted()
+        {
+            TaskCompletedClass taskCompletedClass = new TaskCompletedClass();
+
+            Task result = taskCompletedClass.UseTaskCompletedMethodAsync();
+
+            Assert.Equal(Task.CompletedTask, result);
```

Comment:
This test is potentially brittle as Task computes equality based on reference equality. According to MS docs, Task.CompletedTask does not always return the same instance on repeated calls. Most likely this test would pass, but as it is not guaranteed, we should think of another way to test this case. Maybe it would be sufficient to call Assert.True(result.IsCompletedSuccessfully)

Reply:
Okay. I understand. Thank you